content/wiki/autotest-integration.md

```diff
@@ -109,7 +111,7 @@ It's useful to use the Pretty formatter in autotest to see exactly what's going
 
 If you define an `autotest-all` profile, autotest will use this profile instead when running your Features. You may want to use the Progress formatter for `autotest-all`.
 
-For more information on defining profiles, see [Running Features](/configuration/#running-features).
+For more information on defining profiles, see [Running Features](/running/).
```

Comment:
Change link "[Running Features]" to "Running Cucumber"?

Reply:
Done